<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFormBulletinClass</class>
 <widget class="QDialog" name="DialogFormBulletinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogFormBulletin</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title (*)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Content (*)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="content">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_date">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="end_date">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End date</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_actived">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Actived</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_priority">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>High priority</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To employee</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget { border: 1px solid #ccc; }</string>
   </property>
   <widget class="QRadioButton" name="radio_all">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_deparment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_selection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filter_employee">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="employee_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="employee_widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>349</width>
       <height>109</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="department_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_no">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/close.png</normaloff>../icon/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yes">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>580</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/check.png</normaloff>../icon/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
